analyzer/codechecker_analyzer/analyzers/config_handler.py

```diff
@@ -37,6 +37,18 @@ class CheckerState(Enum):
     enabled = 2
 
 
+def get_compiler_warning_name(checker_name):
+    """
+    Removes 'W' or 'Wno' from the compiler warning name, if this is a
+    compiler warning. Returns None otherwise.
+    """
+    # Checker name is a compiler warning.
+    if checker_name.startswith('W'):
+        return checker_name[4:] if \
+            checker_name.startswith('Wno-') else checker_name[1:]
+    return None
```

Comment:
Functions in Python return `None` by default, conventionally we don't make this explicit.